src/components/ProjectList/css/search.css

```diff
@@ -25,9 +25,17 @@
    margin-top: 40px;
 }
 
+div.errorMessage {
```

Comment:
@SrikanthPynaboina can you make error message in red or yellow background? I think users are used to seeing danger messages in red or yellow. What do you think?